aws-wildfly/pom.xml

```diff
@@ -13,11 +13,25 @@
 
     <name>${project.artifactId}</name>
 
+    <properties>
+        <!-- to guarantee test instance Name uniqueness, supports: https://github.com/wildfly-extras/sunstone/issues/154 -->
+        <timestamp>${maven.build.timestamp}</timestamp>
+        <maven.build.timestamp.format>yyyyMMddHHmmss</maven.build.timestamp.format>
+        <test.instance.name>eapSunstoneInjectInstance-${timestamp}</test.instance.name>
```

Comment:
Can you set (form) the instance name in sunstone.properties? and propagate timestamp into it? I would also consider renaming `timestamp` to `testrun`